imports/plugins/core/orders/client/components/OrderPreviousRefunds.js

```diff
@@ -0,0 +1,98 @@
+import Divider from "@material-ui/core/Divider";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { i18next } from "/client/api";
+
+const styles = (theme) => ({
+  dividerSpacing: {
+    marginBottom: theme.spacing(4),
+    marginTop: theme.spacing(4)
+  },
+  fontWeightSemiBold: {
+    fontWeight: theme.typography.fontWeightSemiBold
+  }
+});
+
+/**
+ * @name OrderPreviousRefunds
+ * @param {Object} props Component props
+ * @returns {React.Component} returns a React component
+ */
+function OrderPreviousRefunds(props) {
+  const { classes, order } = props;
+  const { refunds: unsortedRefunds } = order;
+  // sort refunds by date, newest first, for display purposes
+  const refunds = unsortedRefunds.slice().sort((refundA, refundB) => new Date(refundB.createdAt) - new Date(refundA.createdAt));
+
+  if (Array.isArray(refunds) && refunds.length) {
+    return (
+      <Grid item xs={12}>
+        <Card elevation={0}>
+          <CardHeader
+            title={i18next.t("order.previousRefunds", "Previous Refunds")}
+          />
+          <CardContent>
+            {refunds.map((refund, index) => {
+              const { amount, createdAt, paymentDisplayName, reason } = refund;
+              const formattedDate = moment(createdAt).format("l");
+
+              return (
+                <Grid container key={index}>
+                  <Grid item xs={6} md={6}>
+                    <Grid item xs={12} md={12}>
+                      <Typography paragraph variant="h4">
+                        {formattedDate}
+                      </Typography>
+                      <Typography paragraph variant="body1">
+                        {i18next.t("order.refundedTo", "Refunded to")} <span className={classes.fontWeightSemiBold}>{paymentDisplayName}</span>
+                      </Typography>
+                      <Typography variant="h4">
+                        {i18next.t("order.reasonForRefund", "Reason for refund")}
+                      </Typography>
+                      <Typography paragraph variant="body1">
+                        {reason || i18next.t("order.noRefundReason", "No reason provided")}
+                      </Typography>
+                    </Grid>
+                  </Grid>
+                  <Grid item xs={6} md={6}>
+                    <Grid item xs={12} md={12}>
+                      <Typography paragraph variant="h4" align="right">
+                        {amount.displayAmount}
+                      </Typography>
+                    </Grid>
+                  </Grid>
+                  {index + 1 < refunds.length &&
+                  <Grid item xs={12} md={12}>
+                    <Divider className={classes.dividerSpacing} />
+                  </Grid>
+                  }
+                </Grid>
+              );
+            })}
+          </CardContent>
+        </Card>
+      </Grid>
+    );
+  }
+
+  return null;
```

Comment:
We don't want any empty state text?

Reply:
There is no empty state card in the designs, if there are no previous refunds, it's just not there. 

<img width="1361" alt="Zeplin" src="https://user-images.githubusercontent.com/4482263/62385269-72fc9600-b509-11e9-869f-563b68348e4a.png">